TestAssets/TestPackages/dotnet-portable/Program.cs

```diff
@@ -10,6 +12,20 @@ public class Program
         public static void Main(string[] args)
         {
             Console.WriteLine("Hello Portable World!");
+
+            var depsFile = new FileInfo(GetDataFromAppDomain("FX_DEPS_FILE"));
+            string frameworkVersion = depsFile.Directory.Name;
+
+            Console.WriteLine($"I'm running on shared framework version {frameworkVersion}!");
+        }
+
+        public static string GetDataFromAppDomain(string propertyName)
+        {
+            var appDomainType = typeof(object).GetTypeInfo().Assembly?.GetType("System.AppDomain");
```

Comment:
Nit: since this is only ever called on net Core, you can just use AppContext.GetData. This code is only necessary if you need to support full fx as well